README.md

```diff
@@ -41,3 +41,12 @@ where `./custom-cvc4/build/src/libcvc4.a` and `./custom-cvc4/build/src/parser/li
 
 # Building Tests
  You can build tests with `make test` from the build directory. After you have a full installation, you can build the tests yourself by updating the includes to include the `smt-switch` directory. For example: `#include "cvc4_factory.h"` -> `#include "smt-switch/cvc4_factory.h"`.
+
+## Debug
+The tests currently use C-style assertions which are compiled out in Release mode (the default). To build tests with assertions, please add the `--debug` flag when using `./configure.sh`.
+
+# Python bindings
```

Comment:
Maybe add how to setup cython or have a separate contrib/ script for it. I looked at the .travis.yml to figure out how to make it build.

Reply:
Also, it's probably not a README thing as much as a DEVELOPERS thing, but maybe include a quick note on running your pytests.

Reply:
Addressed with f552b36e710a0a748983f359809d03ed5d49fe58. Does that look good to you? I just put it in the README: in case users want to make sure it installed correctly.